package.json

```diff
@@ -7,11 +7,13 @@
   "dependencies": {
     "@turf/area": "^6.0.1",
     "@turf/helpers": "^6.1.4",
+    "@types/service-worker-mock": "^2.0.1",
     "@types/xml2js": "^0.4.4",
     "axios": "^0.18.1",
     "moment": "^2.24.0",
     "polygon-clipping": "^0.14.3",
     "query-string": "^6.4.2",
+    "service-worker-mock": "^2.0.5",
```

Comment:
Since this is only used by tests, should we add it to `devDependencies`?

Reply:
Yes thanks, moved this to `devDependencies`